packages/search-service/src/search-mapping/schema.ts

```diff
@@ -0,0 +1,31 @@
+import { Document, Model, model, Schema } from 'mongoose';
+
+export const SearchMapSchema: Schema = new Schema({
+  name: String,
+  fields: [{
+    from: String,
+    to: String
+  }],
+  apiConfig: {
+    model: { type: String, enum: ['GRAPHQL', 'REST'] },
```

Comment:
Can you make this more generic?

i.e. No need to ask for GraphQL / ReST. Just ask for request method (GET / POST), and the request body if POST is selected. A GraphQL request is essentially a POST request with a JSON body, so the graphql query can also be provided as `body`.

Reply:
We can simplify it further on the UI using a Postman-like UI for GraphQL requests, while storing it as a normal POST request in the backend.